Remove a lot of mentions of JavaScript versions (#19448)
mdn/content: files/en-us/web/javascript/reference/functions/index.md

### Method definition syntax

> In object literals, you are able to define own methods in a shorter syntax, similar to the getters and setters. See [method definitions](/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions) for more information.

Reviewer: ```suggestion
In object literals, you can define your own methods in a shorter syntax, similar to the getters and setters. See [method definitions](/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions) for more information.
```

Author: "Own methods" as opposed to "inherited methods" is a technical term
